common/production_methods/07_government.txt

```diff
@@ -407,7 +407,128 @@ pm_religious_academia = {
 		}
 	}			
 }			
+pm_no_academic_focus = {
+	texture = "gfx/interface/icons/production_method_icons/unused/no_philosophy_dept.dds"
+}
+pm_military_academy = {
+	texture = "gfx/interface/icons/production_method_icons/unused/military_academy.dds"
+	building_modifiers = {
+		level_scaled = {
+			building_employment_officers_add = 150
+            building_employment_academics_add = 50
+            building_employment_soldiers_add = 250
+		}
+	}
+	building_modifiers = {
```

Comment:
Please only use one building modifier per pm. you can do multiple kinds of effects within one building_modifiers